components/si-labs/CP2105.stanza

```diff
@@ -78,9 +78,8 @@ public pcb-component component :
   assign-landpattern(qfn-landpattern(0.5, 4.0, 24, 0.25, 0.4, [2.7 2.7]))
 
   property(REGIN.power-pin) = PowerPin(min-max(3.0, 5.25))
-  property(VDD.power-pin) = PowerPin(min-max(3.0, 3.6))
-  eval-when has-property?(VDD.net-voltage) :
-    property(VIO.power-pin) = PowerPin(min-max(1.8, max-value(property(VDD.net-voltage))))
+  eval-when has-property?(REGIN.net-voltage) :
+    property(VIO.power-pin) = PowerPin(min-max(1.8, max-value(property(REGIN.net-voltage))))
   eval-when has-property?(VIO.net-voltage) :
```

Comment:
Given that VDD does not need to be tied to VIO because the IO can be operated at a lower voltage so VIO is treated as a power supply input for the CP2105. External devices can be powered from VDD, you might need to support VDD as a supply output as as follows:
```
  ; we assume that REGIN voltage is always copied to VDD (and vice-versa???)
  eval-when has-property?(REGIN.net-voltage) :
    if property(REGIN.net-voltage) > 3.0 : ; this is likely not spec'ed correctly in the datasheet
      property(VDD.power-pin) = PowerPin(min-typ-max(3.3, 3.45, 3.6))
```